lib/Frontend/ModuleInterfaceLoader.cpp

```diff
@@ -736,13 +737,7 @@ class ModuleInterfaceLoaderImpl {
     // Should we attempt to load a swiftmodule adjacent to the swiftinterface?
     bool shouldLoadAdjacentModule = !ctx.IgnoreAdjacentModules;
 
-    // Don't use the adjacent swiftmodule for frameworks from the public
-    // Frameworks folder of the SDK.
-    SmallString<128> publicFrameworksPath;
-    llvm::sys::path::append(publicFrameworksPath,
-                            ctx.SearchPathOpts.getSDKPath(),
-                            "System", "Library", "Frameworks");
-    if (isInSystemFrameworks(modulePath)) {
+    if (isInSystemFrameworks(modulePath, true)) {
```

Comment:
```suggestion
    if (isInSystemFrameworks(modulePath, /*publicFramework=*/true)) {
```
Alternatively, you could use an enum to make this value clearer at the call site.